lib/bark.ex

```diff
@@ -6,21 +6,37 @@ defmodule Bark do
   require Logger
 
   @spec warn(Macro.Env.t(), Keyword.t()) :: :ok
-  def warn(env, opts), do: Logger.warning(parse_message(env, opts))
+  def warn(env, opts), do: Logger.warning(parse_message(env, opts), ansi_color: validate_ansi_color(opts[:ansi_color]))
 
   @spec info(Macro.Env.t(), Keyword.t()) :: :ok
-  def info(env, opts), do: Logger.info(parse_message(env, opts))
+  def info(env, opts), do: Logger.info(parse_message(env, opts), ansi_color: validate_ansi_color(opts[:ansi_color]))
 
   @spec audit(Macro.Env.t(), Keyword.t()) :: :ok
-  def audit(env, opts), do: Logger.notice(parse_message(env, opts))
+  def audit(env, opts), do: Logger.notice(parse_message(env, opts), ansi_color: validate_ansi_color(opts[:ansi_color]))
 
   @spec error(Macro.Env.t(), Keyword.t()) :: :ok
-  def error(env, opts), do: Logger.error(parse_message(env, opts))
+  def error(env, opts), do: Logger.error(parse_message(env, opts), ansi_color: validate_ansi_color(opts[:ansi_color]))
 
   @spec debug(Macro.Env.t(), Keyword.t()) :: :ok
-  def debug(env, opts), do: Logger.debug(parse_message(env, opts))
+  def debug(env, opts), do: Logger.debug(parse_message(env, opts), ansi_color: validate_ansi_color(opts[:ansi_color]))
+
+  @valid_colors [
+    :black, :red, :green, :yellow, :blue, :magenta, :cyan, :white,
+    :light_black, :light_red, :light_green, :light_yellow, :light_blue,
+    :light_magenta, :light_cyan, :light_white,
+    :black_background, :red_background, :green_background, :yellow_background,
+    :blue_background, :magenta_background, :cyan_background, :white_background,
+    :light_black_background, :light_red_background, :light_green_background,
+    :light_yellow_background, :light_blue_background, :light_magenta_background,
+    :light_cyan_background, :light_white_background
+  ]
+
+  defp validate_ansi_color(color) when color in @valid_colors, do: color
```

Comment:
```suggestion
defp validate_ansi_color(color) when is_atom(color), do: if(function_exported?(IO.ANSI, color, 0), do: color, else: nil)
```

Surprised to see logger crashes with an invalid color. idk if I like this better but it does eliminate the need to maintain that list 🤷🏼, but introduces edge cases like `font_8` or `clear` that behave weird. Either way is good with me!

Reply:
On the bright side, I doubt we'd ever have to update/maintain the list. 😄 

Unless new colors are ever invented? 🤔

Reply:
Ah yeah I was a bit sad logger crashes with an invalid color.  I was hoping with https://github.com/smartrent/bark/issues/9 and us moving to a `require Bark`, that some of those inconsistencies in options would be resolved there  😬